sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/surveillance/SurveillanceDashboardView.java

```diff
@@ -39,15 +39,16 @@ public class SurveillanceDashboardView extends AbstractDashboardView {
 
 	public SurveillanceDashboardView() {
 		super(VIEW_NAME);
-
-		dashboardDataProvider = new DashboardDataProvider();
+		dashboardDataProvider = new DashboardDataProvider(NewCaseDateType.class);
 		if (dashboardDataProvider.getDashboardType() == null) {
 			dashboardDataProvider.setDashboardType(DashboardType.SURVEILLANCE);
 		}
 		if (DashboardType.CONTACTS.equals(dashboardDataProvider.getDashboardType())) {
 			dashboardDataProvider.setDisease(FacadeProvider.getDiseaseConfigurationFacade().getDefaultDisease());
 		}
+		dashboardDataProvider.setNewCaseDateType(NewCaseDateType.MOST_RELEVANT);
 		filterLayout = new SurveillanceFilterLayout(this, dashboardDataProvider);
+
```

Comment:
Check the spacing between the filterLayout.
This is how it should be

"filterLayout = new SurveillanceFilterLayout(this, dashboardDataProvider);
		filterLayout.addDateTypeValueChangeListener(e -> {
			dashboardDataProvider.setNewCaseDateType((NewCaseDateType) e.getProperty().getValue());
		});
		dashboardLayout.addComponent(filterLayout);"